src/EmbeddedVsix/EmbeddedVsix.csproj

```diff
@@ -32,28 +38,43 @@
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
     <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <DefineConstants>$(DefineConstants);DEBUG;TRACE</DefineConstants>
```

Comment:
This was the cause of one of the signing-related build issues. This property settings was overwriting the `DefineConstants` value set in the signing targets.

The new SDK-style project don't set this property explicitly in the project file, and presumably the new SDK-style targets handle aggregating with existing values correctly.